include/aws/sdkutils/sdkutils.h

```diff
@@ -18,13 +18,17 @@ enum aws_sdkutils_errors {
     AWS_ERROR_SDKUTILS_GENERAL = AWS_ERROR_ENUM_BEGIN_RANGE(AWS_C_SDKUTILS_PACKAGE_ID),
     AWS_ERROR_SDKUTILS_PARSE_FATAL,
     AWS_ERROR_SDKUTILS_PARSE_RECOVERABLE,
+    AWS_ERROR_SDKUTILS_ENDPOINTS_UNSUPPORTED_RULESET,
+    AWS_ERROR_SDKUTILS_ENDPOINTS_PARSE_FAILED,
+    AWS_ERROR_SDKUTILS_ENDPOINTS_INVALID_ARGUMENT,
```

Comment:
Don't add this, just use `AWS_ERROR_INVALID_ARGUMENT` from aws-c-common.

It's OK to use an error code from a library you depend on